crm_phonecall_timesheet/models/crm_phonecall.py

```diff
@@ -26,51 +26,65 @@ def _timesheet_prepare(self, vals):
         """
         if len(self) > 0:
             self.ensure_one()
-        date = vals.get('date', self.date)
+        date = vals.get(
+            'date',
+            self.date and self.date.strftime(DEFAULT_SERVER_DATE_FORMAT))
```

Comment:
Use `fields.Date.from_string` instead

Reply:
This convert datetime to string, ``` fields.Date.from_string``` convert string to datetime

Reply:
Then use `fields.Date.to_string`